__tests__/commands/pages/upload.test.ts

```diff
@@ -31,16 +35,17 @@ describe('rdme references upload', () => {
   });
 
   afterEach(() => {
+    prompts.override({});
```

Comment:
don't ask me how long it took to figure out that this was necessary to get a couple of tests passing again 🫠